### ISSUE TITLE
error message says -st needed, even when supplied

### ISSUE THREAD

**Issue by childers** [NAL-i5K/GFF3toolkit#5]
I was trying to check out the sequences from a gff and the program stopped with an error, even through I supplied the requested flag:
```shell
$ python -V
Python 2.7.13

$ python ../../bin/gff3_to_fasta.py -f Ofas.scaffolds.fa -st all -g oncfas_OGSv1.2_original.gff -o test 
INFO     Checking gff file (oncfas_OGSv1.2_original.gff)...
INFO     Checking genome fasta (Ofas.scaffolds.fa)...
INFO     Specifying sequence type: (all)...
usage: gff3_to_fasta.py [-h] [-g GFF] [-f FASTA] [-st SEQUENCE_TYPE]
                        [-d DEFLINE] [-o OUTPUT_PREFIX] [-noQC] [-v]

Extract sequences from specific regions of genome based on gff file.
Testing enviroment:
1. Python 2.7

Required inputs:
1. GFF3: specify the file name with the -g argument
2. Fasta file: specify the file name with the -f argument
3. Output prefix: specify with the -o argument

Outputs:
1. Fasta formatted sequence file based on the gff3 file.

Example command: 
python2.7 bin/gff3_to_fasta.py -g example_file/example.gff3 -f example_file/reference.fa -st all -d simple -o test_sequences

optional arguments:
  -h, --help            show this help message and exit
  -g GFF, --gff GFF     Genome annotation file in GFF3 format
  -f FASTA, --fasta FASTA
                        Genome sequences in FASTA format
  -st SEQUENCE_TYPE, --sequence_type SEQUENCE_TYPE
                        Type of sequences you would like to extract: 
                        	"all" - FASTA files for all types of sequences listed below;
                        	"gene" - gene sequence for each record;
                        	"exon" - exon sequence for each record;
                        	"pre_trans" - genomic region of a transcript model (premature transcript);
                        	"trans" - spliced transcripts (only exons included);
                        	"cds" - coding sequences;
                        	"pep" - peptide sequences.
  -d DEFLINE, --defline DEFLINE
                        Defline format in the output FASTA file:
                        	"simple" - only ID would be shown in the defline;
                        	"complete" - complete information of the feature would be shown in the defline.
  -o OUTPUT_PREFIX, --output_prefix OUTPUT_PREFIX
                        Prefix of output file name
  -noQC, --quality_control
                        Specify this option if you do not want to excute quality control for gff file. (default: QC is excuted)
  -v, --version         show program's version number and exit
ERROR    Required field -st missing...
```

**Comment by childers** [comment]
Adding ```-d simple and -o test``` to the command above caused the program to work. There might be an issue in the way the input options are being processed.

**Comment by mpoelchau** [comment]
Resolved via #7. Closing.
